### PR TITLE
1L5gZkHN-login-functionalities

### PR DESCRIPTION
**Pull request by Code-250**
### What does this PR do?

- Allows admin to login and store the login credentials in the local storage

### Description of Task to be completed?
- validate the login form input fields
- store the login credentials in the local storage of the browser
- redirect the admin to the dashboard page if authenticated successfully
- handle error messages

### How should this be manually tested?
- `git clone paste the repository link` in your terminal
- `git checkout ft-login-functionalities-1L5gZkHN` to go to the actual branch to test
- go to the login page then start to test the functionalities
- after meeting all the required validation you will be logged in and your credentials will be saved in your browser's localStorage.

### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

[login-functionalities](https://trello.com/c/1L5gZkHN/19-login-functionalities)

### Screenshots (if appropriate)

image

### Questions:
N/A

### CODE REVIEW

File: js/validations/loginformValidation.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    // show password error message to user
+    passwordErrorMessageElement.innerText = passowrdErrorMessage;
+  }
+  if ((emailErrorMessage === " ") & (passowrdErrorMessage === " ")) {
+    const saveEmail = JSON.stringify(email);
+    const savePassword = JSON.stringify(password);
+    console.log(saveEmail, savePassword);
+    localStorage.setItem("email", saveEmail);
+    localStorage.setItem("password", savePassword);
+    window.location.replace("http://127.0.0.1:5500/admin/admin.html");
```
Comment (ncutixavier) [inline js/validations/loginformValidation.js]: - Change redirect URL
- Add email & password in local storage as an object

**Review by ncutixavier — CHANGES_REQUESTED** [review]

File: js/validations/loginformValidation.js (hunk 1 above)
Comment (Code-250) [reply js/validations/loginformValidation.js]: okay, How do you want me to change the redirect link can I use the links from netlify?

Comment (ncutixavier) [reply js/validations/loginformValidation.js]: Yes

Comment (Code-250) [reply js/validations/loginformValidation.js]: I have collected the requested changes please review them whenever you have time.
thanks

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**